src/components/drawer/index.js

```diff
@@ -68,7 +68,23 @@ class NewDrawer extends PureBaseComponent {
     /**
      * Perform the animation in natively
      */
-    useNativeAnimations: PropTypes.bool
+    useNativeAnimations: PropTypes.bool,
```

Comment:
At first I thought this was added by mistake because we have `...Swipeable.PropTypes` at the top, but it looks like the `...Swipeable.PropTypes` should be removed, not sure if you want to do it in this PR.

Reply:
removed